frontend/src/metabase/list-view/components/ListView/ListViewConfiguration.tsx

```diff
@@ -379,9 +386,9 @@ export const ListViewConfiguration = ({
           data-testid="list-view-preview"
         >
           <Text fw="bold">{t`Preview`}</Text>
-          {firstRow ? (
+          {previewSample ? (
```

Comment:
Why do we only show 1 row in the preview?
It looks quite scarce, especially on a tall screen.
If we showed more rows, there would be a higher chance to get non-empty values for every column.

<img width="3158" height="2592" alt="image" src="https://github.com/user-attachments/assets/3f61c51f-55e6-4a15-a09f-5141ff133b9f" />